packages/leancode_lint/test/lints_test_app/lib/avoid_single_child_in_multi_child_widget_test.dart

```diff
@@ -0,0 +1,194 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+class ColumnIsMarked extends StatelessWidget {
+  const ColumnIsMarked({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    // expect_lint: avoid_single_child_in_multi_child_widgets
+    return Column(
+      children: [
+        Container(),
+      ],
+    );
+  }
+}
+
+class RowIsMarked extends StatelessWidget {
+  const RowIsMarked({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    // expect_lint: avoid_single_child_in_multi_child_widgets
+    return Row(
+      children: [
+        Container(),
+      ],
+    );
+  }
+}
+
+class FlexIsMarked extends StatelessWidget {
+  const FlexIsMarked({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    // expect_lint: avoid_single_child_in_multi_child_widgets
+    return Flex(
+      direction: Axis.horizontal,
+      children: [
+        Container(),
+      ],
+    );
+  }
+}
+
+class WrapIsMarked extends StatelessWidget {
+  const WrapIsMarked({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    // expect_lint: avoid_single_child_in_multi_child_widgets
+    return Wrap(
+      children: [
+        Container(),
+      ],
+    );
+  }
+}
+
+class ZeroChildrenIsNotMarked extends StatelessWidget {
+  const ZeroChildrenIsNotMarked({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Wrap();
```

Comment:
Could be a separate lint, but maybe we want to disallow no children as well?

Reply:
I thought about it, but figured that there are so many widgets that dont make sense when used with no children/child. Also I would assume it happens way less often, while single child in Column/Row happens quite often by accident. So yeah, if we want a no children/child lint it should be separate